### DISCUSSION TITLE
Wrong Owner off heads

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#2825] [Enhancement]
The management of heads seems to be wrong.. or just the showed once

If you rightclick a placed head it teöös you:

Owner: (YOURNAME)...
looking at the langfiles show's me:
skullOwner: '!actionbar!&7Owner:&r [playerName]'

Well no wonder that it shows just the name of the rightklicker no matter who is the real owner...

i hope this can be fixed  :)

![image](https://user-images.githubusercontent.com/12080017/75163945-58e20980-5720-11ea-8b6e-d7d5636886df.png)

**Comment by Zrips** [comment]
In general this shows head skin owner which if possible will be retrieved from database by stored UUID in this head. If you have offline server then it might be somewhat funky and show completely different user name as owner of this head. We cant really store data in blocks, atleast not in older then 1.14 servers. Extra work is needed to have better recognition of a heads.

**Comment by CloudeLecaw** [comment]
wait wait wait..
First of all Lululand is no offline Server and the secound thing is Lululand is a 1.15.2 Server.

Why you thought im talking about an offline older 1.14 server? :3

**Comment by LogGits** [comment]
@CloudeLecaw can you send the /blocknbt data of that headblock. I have noticed that for example, a playerhead with no data defaults to the name of the person who rightclicks the head.

When a head is retrieved via `/cmi head`, rightclicking it will yield the owner of the head.
What did you use to get this head and send us the /blocknbt.

**Comment by CloudeLecaw** [comment]
i cant do this - Our Serversowner Blueeyestar is the only one that can execute this comment.
may he also can tell you where the heads are from.

**Comment by Blueeyestar** [comment]
I get the heads over this plugin:

https://www.spigotmc.org/resources/head-database.14280/
